### ISSUE TITLE
None linear levels

### ISSUE THREAD

**Issue by sciencebloke50** [TheGameCreators/GameGuruRepo#2326]
hi i am unable to attach mutliple levels to one main level.

i have a main map world but want to jumpt to an underground level and at the end of that jump back to where i was. this was apparently possible yet only one level attaches to one level making it linear at present. is this correct or is this an error?

**Comment by ZakJudges** [comment]
Hi @sciencebloke50, you can place a winzone down and change its settings to go to a specific level when the player enters it. Note the level that you want to jump to must be added to your storyboard.
![Screenshot 2022-02-28 120507](https://user-images.githubusercontent.com/32545391/155980705-70a878ff-3fa2-4337-adef-f22ffee7b134.png)

**Comment by sciencebloke50** [comment]
Yes i have done that. but when i have placed the win zone in and linked it to the level when testing the full game it does not jump.

Also my main question which was answered yes by lee online
Is being able to jump back to the same level a sort of link system ..

That appears to be impossible as there is no way to add a complex linking system from one level to another. I can only go fowards having 6 cave levels attached to a main outdoor level is not possible. Yet i was told it will be
________________________________

**Comment by ZakJudges** [comment]
@sciencebloke50 Did you try it in standalone? If you are testing from the editor, I believe you just get a message displayed that says jump to: level name, but it won't load the new level since you are still editing the current one.

I'll keep this issue open as an enhancement, to remember the current state of the level so it can be restored when you come back to it.

If you tried to link levels together in standalone and it did not work, then it sounds like a bug, and we can raise a new issue for it.

**Comment by Necrym59** [comment]
Tick the Standalone Test All Levels option under play game for it to work in test mode
